templates/profiles.html

```diff
@@ -479,48 +560,29 @@ <h3>Create a profile</h3>
                                             <label class="label">timeout</label>
                                         </div>
                                         <div class="field-body">
-                                            <div class="field">
+                                            <div class="field ">
                                                 <div class="control">
                                                     <input class="input is-small" type="number" x-model="executor.timeout">
                                                 </div>
                                             </div>
                                         </div>
                                     </div>
-                                    <template x-for="(cleanup, index) of executor.cleanup">
```

Comment:
We'll want to add this cleanup command field back the way it was. The cleanup command is now an array of strings from the API, so having a single input would likely throw errors at some point. If you check out the abilities page and click on one of the rows, you'll see that logic in action.

While you have this PR up, I noticed a small bug on the abilities page with the edit modal. When I clicked on "+ Cleanup Command", it refreshed the page so all the tabs close out. This is because that add button is acting like a form submitter, so if you add `type="button"` to that `<button>` tag it should prevent that. Once this field is reverted you may run into the same issue on this page, so figured it'd make sense to tackle that now.

Reply:
Added in latest push, hopefully caught them all!